internal/bundle/apps_test.go

```diff
@@ -0,0 +1,80 @@
+package bundle
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/databricks/cli/internal"
+	"github.com/databricks/cli/internal/acc"
+	"github.com/databricks/cli/libs/env"
+	"github.com/databricks/databricks-sdk-go/service/apps"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAccDeployBundleWithApp(t *testing.T) {
+	ctx, wt := acc.WorkspaceTest(t)
+	uniqueId := uuid.New().String()
+	appId := fmt.Sprintf("app-%s", uuid.New().String()[0:8])
+	nodeTypeId := internal.GetNodeTypeId(env.Get(ctx, "CLOUD_ENV"))
+	instancePoolId := env.Get(ctx, "TEST_INSTANCE_POOL_ID")
+
+	root, err := initTestTemplate(t, ctx, "apps", map[string]any{
+		"unique_id":        uniqueId,
+		"app_id":           appId,
+		"node_type_id":     nodeTypeId,
+		"spark_version":    defaultSparkVersion,
+		"instance_pool_id": instancePoolId,
```

Comment:
The example doesn't need these last 3. You could make the job serverless and trigger a notebook.

Reply:
Unfortunately not all our environments have enabled serverless ```
    helpers.go:238: [databricks stdout]: Error: cannot create job: One of job_cluster_key, new_cluster, or existing_cluster_id must be specified. Serverless compute for workflows is not enabled in the workspace.
    ```
    
    Was planning to look at it separately

Reply:
Thanks!